unlock-app/src/components/interface/RefundButton.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { ethers } from 'ethers'
+import withConfig from '../../utils/withConfig'
+
+interface RefundButtonProps {
+  accountAddress: string
+  lockAddress: string
+  config: {
+    providers: any
+    externalRefundContractAddress: string
+    env: 'dev' | 'test' | 'staging' | 'prod'
+  }
+}
+interface RefundButtonState {
+  refundInitiated: boolean
+}
+
+const lockAddressesByEnv = {
+  dev: '0x0AAF2059Cb2cE8Eeb1a0C60f4e0f2789214350a5',
+  test: '0x0AAF2059Cb2cE8Eeb1a0C60f4e0f2789214350a5',
+  staging: '',
```

Comment:
Tried parsing the input of [the rinkeby external refund contract](https://rinkeby.etherscan.io/address/0x3e81845518f775eebfc1a4fb62123113ff58159a) to see what lock it was created against, and it looks like the address I pulled out wasn't a deployed lock, so I may have done something wrong.

We should hold off on merging this until we know what value to use for staging.